.github/workflows/build_publish.yml

```diff
@@ -68,23 +77,3 @@ jobs:
           rm transifex.sh
         env:
           TX_TOKEN: ${{ secrets.TX_TOKEN }}
-
-  trigger-workflow:
-    name: 🔁 Trigger workflow to deploy to Docker env
```

Comment:
Can we reuse the trigger to bahmni-docker. Lets raise an event for now, Once we bundle all the omods and frontend, we can enable from docker repository for auto deployment